### DISCUSSION TITLE
Why I can't run my manim?

### DISCUSSION THREAD

**Discussion by CharlieGao2009** [3b1b/manim#2098] [Q&A]
When I run `manimgl` in command prompt, the following error occurs:
```
Traceback (most recent call last):
  File "<frozen runpy>", line 198, in _run_module_as_main
  File "<frozen runpy>", line 88, in _run_code
  File "C:\Users\gao\AppData\Roaming\Python\Python311\Scripts\manimgl.exe\__main__.py", line 4, in <module>
  File "C:\Users\gao\AppData\Roaming\Python\Python311\site-packages\manimlib\__main__.py", line 6, in <module>
    from manimlib import __version__
ImportError: cannot import name '__version__' from 'manimlib' (C:\Users\gao\AppData\Roaming\Python\Python311\site-packages\manimlib\__init__.py)
```
Could anybody help me solve this? Thank you very much :D

**Comment by Pluser456** [comment]
I have the same problem with you. Have you solved it now?

**Comment by EgorusAbobus434** [comment]
Same

**Comment by mitkonikov** [comment]
This is because 3b1b's Manim latest release is 1.6.1 on Apr 13, 2022 and it only supports up to Python 3.10.
There are also issues, where on that release, you will need to have numpy <= 1.24 version.
However, you can clone the repository and follow the instructions for installing it from the repository instead from PyPI.
Be aware that the repository is not a stable release.

Briefly, the instructions:
1. Clone the 3b1b/manim repo (no need to fork it)
2. Install the dependencies (ffmpeg, latex)
3. `pip install -r requirements.txt`
4. Inside the repository run `pip install -e .`
5. Try it out with `manimgl example_scenes.py OpeningManimExample`

For me, the current repository even works in Python 3.12.7.
